packages/components/src/MessageBar/MessageBar.tsx

```diff
@@ -247,10 +266,9 @@ export const MessageBar = styled(MessageBarLayout).attrs(
   })
 )`
   ${simpleLayoutCSS}
-
   align-items: center;
-  background: ${({ intent, theme: { colors } }) =>
-    intent === 'critical' ? colors.criticalAccent : colors.neutralAccent};
+  ${({ intent, theme: { colors } }) =>
+    intent ? backgroundColor : `background: ${colors.informAccent};`}
```

Comment:
Rather than having the default fallback here we should do it in `attrs` above:

```suggestion
  ${background}
```